### PR TITLE
landscaper 1.0.10 (new formula)

### PR DESCRIPTION
**Pull request by hoesler**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Used the kubernetes-helm formula as template

### CODE REVIEW

File: Formula/landscaper.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+class Landscaper < Formula
+  desc "Manage the application landscape in a Kubernetes cluster"
+  homepage "https://github.com/Eneco/landscaper"
+  url "https://github.com/Eneco/landscaper.git",
+      :tag => "1.0.7",
```
Comment (ilovezfs) [inline Formula/landscaper.rb]: Does building from https://github.com/Eneco/landscaper/archive/1.0.7.tar.gz work?

File: Formula/landscaper.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      system "make", "bootstrap"
+
+      # Make binary
+      system "make", "build"
+      bin.install "build/landscaper"
+    end
+  end
+
+  test do
+    version_output = shell_output("#{bin}/landscaper version 2>&1")
```
Comment (ilovezfs) [inline Formula/landscaper.rb]: This should be a test that exercises, and verifies, the software's functionality once installed.

**Review by ilovezfs — DISMISSED** [review]

File: Formula/landscaper.rb (hunk 1 above)
Comment (hoesler) [inline Formula/landscaper.rb]: Is building from an archive preferred over a git checkout?

File: Formula/landscaper.rb (hunk 2 above)
Comment (hoesler) [inline Formula/landscaper.rb]: Agreed, but currently landscaper has only two commands: apply and version. Apply, however, needs a connection to a kubernetes cluster. I don't think that this is a feasible assumption.

File: Formula/landscaper.rb (hunk 1 above)
Comment (ilovezfs) [inline Formula/landscaper.rb]: Yes

File: Formula/landscaper.rb (hunk 2 above)
Comment (ilovezfs) [inline Formula/landscaper.rb]: You can simulate it or check for the specific failure message.

Comment (hoesler) [inline Formula/landscaper.rb]: Okay, I am now running `landscaper apply 2>&1` and check for a certain message in stderr while ignoring the exit status. Is that okay?

File: Formula/landscaper.rb (hunk 1 above)
Comment (hoesler) [inline Formula/landscaper.rb]: Found out that the Makefile uses git to set some variables [like VERSION](https://github.com/Eneco/landscaper/blob/master/Makefile#L9).
So I think, we should build from a git checkout.

**Comment by JCount** [comment]
@BrewTestBot test this please

**Comment by JCount** [comment]
@hoesler Thank you a lot for the new formula! Sorry it took so long to get it merged. :fireworks:

**Pull request closed without merging**